layouts/partials/nav_links.html

```diff
@@ -0,0 +1,71 @@
+<nav class="next-previous-links">
+    {{ $sectionParentPages := .CurrentSection.Parent.Pages }}
+    {{ $currentSection := .CurrentSection.Title }}
+
+
+    <sl-divider style="--spacing: 4rem 0rem;">></sl-divider>
```

Comment:
no inline style

there is a better way to achieve this.... you even have css defined in the .scss file

Reply:
resolved with styling added to sl-divider selector, nested under next-previous-links class in .scss